app/models/repository_date_value.rb

```diff
@@ -14,7 +14,8 @@ def self.add_filter_condition(repository_rows, join_alias, filter_element)
     parameters = filter_element.parameters
     case filter_element.operator
     when 'today'
-      repository_rows.where("#{join_alias}.data::date >= ?", Time.zone.now.beginning_of_day)
+      repository_rows.where("#{join_alias}.data::date >= ? AND #{join_alias}.data::date < ?",
```

Comment:
May be this will be more logical?:

repository_rows.where("#{join_alias}.data::date = ?", Time.zone.now.to_date)